tests/test_imputation.py

```diff
@@ -25,6 +25,10 @@
 import iqrs_wrangler as lambda_iqrs_wrangler_function
 import recalculate_means_wrangler as lambda_recalc_wrangler_function
 
+incomplete_runtime_variables = {
```

Comment:
This is for value Error. I just wondered if instead you passed None to the test and then have an if that chooses whether to call value_error with or without passing in runtime variables might be better?